packages/discord.js/src/structures/ContextMenuCommandInteraction.js

```diff
@@ -28,10 +28,10 @@ class ContextMenuCommandInteraction extends CommandInteraction {
     this.targetId = data.data.target_id;
 
     /**
-     * The type of the target of the interaction; either USER or MESSAGE
+     * The type of the target of the interaction; either `.User` or `.Message`
      * @type {ApplicationCommandType}
```

Comment:
Shouldn't the type be `ApplicationCommandType.User|ApplicationCommandType.Message`?